node/src/chain_spec.rs

```diff
@@ -48,8 +100,35 @@ impl Extensions {
 	}
 }
 
+/// Function to generate accounts given a mnemonic and a number of child accounts to be generated
+/// Defaults to a default mnemonic if no mnemonic is supplied
+pub fn generate_accounts(mnemonic: Option<String>, num_accounts: u32) -> Vec<AccountId> {
+	let parent_mnemonic = mnemonic.unwrap_or(
+		"bottom drive obey lake curtain smoke basket hold race lonely fit walk".to_string(),
+	);
+	let childs = derive_pairs_from_mnemonic::<ecdsa::Public>(&parent_mnemonic, num_accounts);
+	debug!("Account Generation");
+	let mut accounts: Vec<AccountId>;
+	accounts = childs
+		.iter()
+		.map(|par| {
+			let account = get_account_id_from_pair::<ecdsa::Public>(par.clone());
+			debug!(
+				"private_key {} --------> Account {:x?}",
+				sp_core::hexdisplay::HexDisplay::from(&par.clone().seed()),
+				account
+			);
+			account
+		})
+		.flatten()
+		.collect();
+	accounts.push(AccountId::from_str("6Be02d1d3665660d22FF9624b7BE0551ee1Ac91b").unwrap());
```

Comment:
Gerald should not be added here. The function should follow the comment description you wrote.